tools/devtool

```diff
@@ -706,6 +725,11 @@ cmd_build() {
     # accessible location.
     [ $ret -eq 0 ] && {
         cargo_bin_dir="$CARGO_TARGET_DIR/$target/$profile"
+
+        # Seccompiler has a different build folder, so copy its contents here.
+        seccompiler_bin_dir="$CARGO_SECCOMPILER_TARGET_DIR/$target/$profile"
```

Comment:
I think we really need to think of another solution,  I will try to figure out something and come back.

Reply:
Nice call-outs. All these problems you encountered above are rooted in the fact that we copy the seccompiler build directory to Firecracker's default target directory. I thought that this would be more intuitive, since all binaries would be placed in the same location.

However, I think it causes more side-effects than benefits.

Removing the copy command should fix those problems, so I'll go ahead and do it

Reply:
> Shouldn t the secompiler target folder get deleted after this?

not really, it is a cache mechanism to make sure that we don't rebuild it unnecessarily